src/test/java/com/avioconsulting/mule/vault/util/VaultContainer.java

```diff
@@ -32,44 +33,36 @@ public class VaultContainer implements TestRule {
     public final static String CLIENT_TRUSTSTORE = SSL_DIRECTORY + File.separator + "truststore.jks";
 
     public final static String CONTAINER_STARTUP_SCRIPT = "/vault/config/startup.sh";
-    public final static String CONTAINER_CONFIG_FILE = "/vault/config/config.json";
-    public final static String CONTAINER_OPENSSL_CONFIG_FILE = "/vault/config/ssl/libressl.conf";
+    public final static String CONTAINER_CONFIG_FILE = "/vault/config/config.hcl";
+    public final static String CONTAINER_OPENSSL_CONFIG_FILE = "/vault/config/libressl.conf";
     public final static String CONTAINER_SSL_DIRECTORY = "/vault/config/ssl";
     public final static String CONTAINER_CERT_PEMFILE = CONTAINER_SSL_DIRECTORY + "/vault-cert.pem";
     public final static String CONTAINER_CLIENT_CERT_PEMFILE = CONTAINER_SSL_DIRECTORY + "/client-cert.pem";
     public final static String CONTAINER_WEB_POLICY_FILE = "/vault/config/web_policy.hcl";
 
-    private final GenericContainer container;
+    private final GenericContainer<?> container;
 
     private String unsealKey;
     private String rootToken;
     private boolean kv2Enabled = false;
 
     public VaultContainer() {
-        container = new GenericContainer("vault:1.1.0")
+        container = new GenericContainer(DockerImageName.parse("vault:1.10.4"))
             .withClasspathResourceMapping("/container_config/startup.sh", CONTAINER_STARTUP_SCRIPT, BindMode.READ_ONLY)
-            .withClasspathResourceMapping("/container_config/config.json", CONTAINER_CONFIG_FILE, BindMode.READ_ONLY)
+            .withClasspathResourceMapping("/container_config/config.hcl", CONTAINER_CONFIG_FILE, BindMode.READ_ONLY)
             .withClasspathResourceMapping("/container_config/libressl.conf", CONTAINER_OPENSSL_CONFIG_FILE, BindMode.READ_ONLY)
             .withClasspathResourceMapping("/policies/web_policy.hcl", CONTAINER_WEB_POLICY_FILE, BindMode.READ_ONLY)
-            .withEnv("VAULT_VERSION", "1.1.0")
```

Comment:
Should we extract the version to a static value and use it for this and the container version to simplify future upgrades?

Reply:
I made the update in the latest commit. Now uses VAULT_VERSION static variable for this and container references